src/wp-admin/includes/post.php

```diff
@@ -1961,15 +1961,75 @@ function wp_create_post_autosave( $post_data ) {
 		 * @param array $new_autosave Post array - the autosave that is about to be saved.
 		 */
 		do_action( 'wp_creating_autosave', $new_autosave );
-
+		wp_autosave_post_revisioned_meta_fields( $new_autosave );
 		return wp_update_post( $new_autosave );
 	}
 
 	// _wp_put_post_revision() expects unescaped.
 	$post_data = wp_unslash( $post_data );
 
 	// Otherwise create the new autosave as a special post revision.
-	return _wp_put_post_revision( $post_data, true );
+	$revision = _wp_put_post_revision( $post_data, true );
```

Comment:
This function can return a WP_Error. We should have some error handling here.

Reply:
👍🏼 added